crates/spirv-std/src/ray_tracing.rs

```diff
@@ -0,0 +1,9 @@
+//! Ray-tracing data types
+
+/// An acceleration structure type which is an opaque reference to an
+/// acceleration structure handle as defined in the client API specification.
+#[spirv(acceleration_structure_khr)]
+#[derive(Copy, Clone)]
+pub struct AccelerationStructureKhr {
+    pub(crate) _private: (),
```

Comment:
I'm a little worried about weird edge cases with rustc thinking it's a ZST, I think I'd prefer this to be a `u32` like all the other fake types

Reply:
Shouldn't it be treated as a ZST though? The spec says it has no defined size or bit pattern. I'm fine with changing it though.

Reply:
"no defined size" is very different than "zero size" - it's the difference between `[T]` and `[T; 0]`. Images are also "no defined size or bit pattern", and we use u32 for those.